### ISSUE TITLE
Track User vs Internal Error in query log

### ISSUE THREAD

**Issue by nathanielc** [influxdata/influxdb#14299]
Once https://github.com/influxdata/flux/issues/1364 is complete we will be able to check error codes of errors produced by Flux and track them in the query log.

At the very least we should track user vs internal errors in the query log, but we could potentially store the error code itself in the query log and defer the decision of user vs internal to reading the database.

**Comment by nathanielc** [comment]
This means that we need to explicitly translate the Flux error codes to InfluxDB error codes and then consume those codes inside the log data.

DOD:

We can distinguish between users and internal errors in the query log database.

**Comment by jsternberg** [comment]
I tested this out and we're going to need a separate issue to convert a lot of the Flux errors to use the new error library. The places where it does exist get wrapped in other unrecognized error types and so almost everything shows up as an internal error anyway.
